src/auth/authAPI.tsx

```diff
@@ -0,0 +1,54 @@
+import Axios from './axios';
+import Token from './token';
+
+enum ROUTES {
+  LOGIN = '/api/v1/user/login/',
+  SIGNUP = '/api/v1/user/signup/',
+  REFRESH = '/api/v1/user/login/refresh/',
+}
+
+interface LoginRequest {
+  readonly email: string;
+  readonly password: string;
+}
+
+export const login = async (user: LoginRequest) =>
```

Comment:
Huh, did not know you can have multiple exports like this, always though you could only have one.

Reply:
You can only have one `export default ______`, you can have as many exports as you want though. The difference affects importing: 
`import Thing from 'thing'` will import the default export
`import { login } from './authAPI.tsx` will import this "named export"